pyext/src/analysis_trajectories.py

```diff
@@ -80,7 +125,8 @@ def __init__(self,
         self.Validation = self.manager.dict()
 
         # Sample stat file
-        stat_files = np.sort(glob.glob(self.out_dirs[0]+'/stat.*.out'))
+        stat_files = glob.glob(os.path.join(self.out_dirs[0], 'stat.*.out'))
+        stat_files.sort()
```

Comment:
`stat_files = sorted(glob.glob(...))` is cleaner and possibly faster than in-place `stat_files.sort()`.

Reply:
using ```sorted()``` now instead of the in-place ```.sort()```